.changeset/happy-donkeys-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/typography': minor
+---
+
+Adds support for `spacing` prop
```

Comment:
Doesn't look like this was done in the typography package. Did you mean for this changeset to be for `Tooltip`?

Reply:
Good catch, thank you